charts/tidb-cluster/templates/tidb-initializer-job.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if or .Values.tidb.passwordSecretName .Values.tidb.initSql }}
+{{- if or .Values.tidb.permitHost .Values.tidb.passwordSecretName .Values.tidb.initSql }}
```

Comment:
I think the initializer cannot work until the `passwordSecretName` is set. So the `permitHost` should be after `passwordSecretName`.